Makefile

```diff
@@ -272,6 +275,16 @@ kind-down: $(KIND)
 	rm -f $(REPO_ROOT)/example/provider-local/base/kubeconfig
 	rm -rf dev/local-backupbuckets
 
+kind2-up: $(KIND)
+	$(KIND) create cluster --name gardener-local2 --config $(REPO_ROOT)/example/gardener-local/kind/cluster2-$(KIND_ENV).yaml --kubeconfig $(GARDENER_LOCAL2_KUBECONFIG)
+	docker exec gardener-local2-control-plane sh -c "sysctl fs.inotify.max_user_instances=8192" # workaround https://kind.sigs.k8s.io/docs/user/known-issues/#pod-errors-due-to-too-many-open-files
+	cp $(GARDENER_LOCAL2_KUBECONFIG) $(REPO_ROOT)/example/provider-local/seed2/kubeconfig
+	kubectl apply -k $(REPO_ROOT)/example/gardener-local/calico --server-side
```

Comment:
This is almost the same like `kind-up`. Can we reuse it and just change the name of the kind cluster?